docs/reverse_proxy.rst

```diff
@@ -85,6 +85,28 @@ Let's assume that we expect clients to connect to our server at
           </Location>
       </VirtualHost>
 
+* HAProxy::
+
+      frontend https
+        bind 0.0.0.0:443 v4v6 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
+        bind :::443 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
+        
+        # A standard website
+        acl www hdr(host) -i www.example.com
+        use_backend www if www
+        
+        # Matrix-Synapse
+        acl matrix hdr(host) -i matrix.example.com
+        use_backend matrix if matrix
+        
+      frontend matrix-synapse
```

Comment:
`matrix-federation` might be a better name

Reply:
Ok!